students/adam_wulw/lesson_05_lists/swap_min_max.py

```diff
@@ -0,0 +1,16 @@
+input_str = input()
+input_list = input_str.split()
+
+index_of_max = 0
+index_of_min = 0
+
+for i in range(0, len(input_list)):
+    if int(input_list[i]) > int(input_list[index_of_max]):
```

Comment:
You can remove int(...) in every line.
You can user input_list = [int(item) for i in input_list] and later use it